src/main/java/world/bentobox/bentobox/util/Util.java

```diff
@@ -605,4 +606,22 @@ public static boolean isInteger(@NonNull String nbr, boolean parse) {
         // Everything's green!
         return true;
     }
+
+    /**
+     * Get a UUID from a string. The string can be a known player's name or a UUID
+     * @param nameOrUUID - name or UUID
+     * @return UUID or null if unknown
+     */
+    @Nullable
+    public static UUID getUUID(@NonNull String nameOrUUID) {
```

Comment:
Good job there! I was afraid that we could no longer use player names in admin commands with your change, but you proved me wrong.